pkg/notes/service/createBookmark.ts

```diff
@@ -21,6 +32,8 @@ export class CreateBookmarkService {
       return Result.err(new Error('bookmark has already created'));
     }
 
-    return await this.bookmarkRepository.create({ noteID, accountID });
+    await this.bookmarkRepository.create({ noteID, accountID });
+
+    return Result.ok(Option.unwrap(note));
```

Comment:
`bookmarkRepository.create` が返すエラーが破棄されてしまっていそうです. `Result.map` で成功時に `note` が反映されるようにしてみてはどうでしょうか.
```suggestion
    const creation = await this.bookmarkRepository.create({ noteID, accountID });

    return Result.map(() => Option.unwrap(note))(creation);
```

Reply:
`8efaf40`で修正しました．